src/sentry/event_manager.py

```diff
@@ -2288,6 +2306,18 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects):
                 EventPerformanceProblem(event, performance_problems_by_hash[problem_hash]).save()
 
 
+@metrics.wraps("performance.performance_issue.should_create_group")
+def should_create_group(client: RedisCluster, grouphash: str) -> bool:
+    times_seen = client.incr(grouphash)
+    if times_seen >= GROUPHASH_IGNORE_LIMIT:
+        client.delete(grouphash)
+        return True
+    else:
+        metrics.incr("performance.performance_issue.ignored")
```

Comment:
We could tag this with `time_seen`. That way, we can roughly estimate how many groups are never being created by subtracting the number of issues that pass this filter from the number of issues seen once (maybe we should add a separate metric to track things we don't ignore?)

Also worth setting the sample rate to 1.0 here.